File: src/main/webapp/js/service/PlayerService.js
app.service("playerService",function ($http) {
    this.findAll = function () {
        return $http.get("../player/query.do");
    }
    this.delete = function (ids) {
        return $http.post("../player/delete.do",ids);
    }
    this.findSchool = function () {
        return $http.get("../player/findSchool.do");
    }
    this.findTeam = function () {
        return $http.get("../player/findTeam.do");
    }
    this.save = function (entity) {
        return $http.post("../player/save.do",entity);
    }
})

File: src/main/java/com/ln/service/PlayerService.java
package com.ln.service;

import com.ln.entity.Player;
import com.ln.entity.School;
import com.ln.entity.Team;

import java.util.List;

public interface PlayerService {
    //查询
    List<Player> getAll();

    void delete(Long[] ids);

    List<School> findSchool();

    List<Team> findTeam();

    void save(Player player);

    Player getOneByName(String pname);
}
